docs/static/_redirects

```diff
@@ -28,3 +28,11 @@
 */wp-login.php        /    301!
 */wlwmanifest.xml     /    301!
 /xmlrpc.php           /    301!
+
+# -----------------------------------------------------------------------------
+# Redirects relating to github asset downloads
+# -----------------------------------------------------------------------------
+
+/downloads/edge/*       https://opa-releases.s3.amazonaws.com/edge/:splat 200
+/downloads/latest/*     https://github.com/open-policy-agent/opa/releases/latest/download/:splat 200
```

Comment:
Previously, we had used a hugo template to render this.

Now, we're making use of a github feature, where https://github.com/open-policy-agent/opa/releases/latest/download/opa_darwin_arm64_static is automatically redirected to the latest release's asset